api/v1alpha1/operatorpipeline_types.go

```diff
@@ -30,6 +30,10 @@ type OperatorPipelineSpec struct {
 
 	// OperatorPipelinesRelease is the Operator Pipelines release (version) to install.
 	OperatorPipelinesRelease string `json:"operatorPipelinesRelease,omitempty"`
+
+	// GitHubToken is the token that will be used by the pipeline to submit a PR to Red Hat
+	//+kubebuilder:validation:Required
```

Comment:
From the pipeline doc, I feel this should be optional if the partners want to test the certification pipeline w/o opening new PR. I might be wrong.

Reply:
Isn’t the github token supposed to be a secret ? putting it in the cr would expose it ?

Reply:
@samira-barouti This is the confusing part, _this_ GH token as I understand it is to raise the PR to the Red Hat repo, the docs also elude to _another_ token that is optional for pinned digest. I feel anything in the CR of this operator should be required, since the pipeline requires these values.

@skattoju This is the part I am not sure about, how can this operator _create_ a secret without it being provided. Is there a way to say the value in the CR is a secret and not a plan text string? I'll look into that.

Reply:
Don't think we should be creating the secret.. just checking if its there based on the secret name provided..

Reply:
My understanding is that this operator's intent is to create all the required assets necessary for the pipeline to run, that way a cluster admin does not have to create anything manually. All they have to do is install the operator and create the CR.

Reply:
We only need to validate the secret exists so I have changed the code to do that and will resolve this comment.